pkgs/top-level/all-packages.nix

```diff
@@ -28655,4 +28655,5 @@ in
 
   psftools = callPackage ../os-specific/linux/psftools {};
 
+  ytt = callPackage ../development/tools/ytt {};
```

Comment:
Please put this under the tools headline and not at the end of everything.

Reply:
Didn't see the sections on the file, but I now see the file-based sorting. thanks.